src/cascadia/TerminalSettingsModel/AIConfig.cpp

```diff
@@ -18,7 +18,44 @@ static constexpr std::wstring_view PasswordVaultAIKey = L"TerminalAIKey";
 static constexpr std::wstring_view PasswordVaultAIEndpoint = L"TerminalAIEndpoint";
 static constexpr std::wstring_view PasswordVaultOpenAIKey = L"TerminalOpenAIKey";
 static constexpr std::wstring_view PasswordVaultGithubCopilotAuthToken = L"TerminalGithubCopilotAuthToken";
-static constexpr std::wstring_view PasswordVaultGithubCopilotRefreshToken = L"TerminalGithubCopilotRefreshToken";
+static constexpr std::wstring_view PasswordVaultGithubCopilotRefreshToken = L"TerminalGithubCopilotRefreshToken";
+static constexpr std::wstring_view AzureOpenAIPolicyKey = L"AzureOpenAI";
+static constexpr std::wstring_view OpenAIPolicyKey = L"OpenAI";
+static constexpr std::wstring_view GitHubCopilotPolicyKey = L"GitHubCopilot";

```

Comment:
I suggest adding a note that with implementation if new ones the adml and policy help (in rekease download) needs to be updated.